import_3dm/converters/render_mesh.py

```diff
@@ -73,10 +75,26 @@ def import_render_mesh(context, ob, name, scale, options):
 
         fidx = fidx + len(m.Vertices)
         vertices.extend([(m.Vertices[v].X * scale, m.Vertices[v].Y * scale, m.Vertices[v].Z * scale) for v in range(len(m.Vertices))])
-    
+        coords.extend([(m.TextureCoordinates[v].X, m.TextureCoordinates[v].Y) for v in range(len(m.TextureCoordinates))])
+   
     mesh = context.blend_data.meshes.new(name=name)
     mesh.from_pydata(vertices, [], faces)
 
+
+    if(mesh.polygons and coords):
+        # todo: 
+        # * check for multiple mappings and handle them
+        # * exclude unmapped objects 
+        mesh.uv_layers.new()
+
+        uvl = mesh.uv_layers.active.data[:]
+        for l in mesh.loops:
+            uvl[l.index].uv = coords[l.vertex_index]
```

Comment:
Hmm, maybe the indices here don't agree with what we think we have. I'd suggest double-checking with a mapping on a simple plane.

Also check the vertex count of one box import.

Reply:
i noticed that. the indices in the loop seem off, but they're actually correct. lets take the box example: i had a test box that had two sides split in triangles, the rest came in as 4 sided polygons. that's 4*3+4*4=28 sets of uv coordinates in the face loops, while using 24 vertices. mesh.uv_layers.new() creates these 28 data sets as needed, but it indeed wouldn't hurt to check list lengths to confirm.

Reply:
@jesterKing: I added the checks and tested it again. The output data for a simple plane and box is as follows. the imprecision on the cube is well within the rhino unit precision and comes from the rhino side (doublechecked through rhino3dm)

```simple planar mapped quad

faces:
[[0, 1, 2, 3]]
coords:
[(0.0, 0.0), (1.0, 0.0), (1.0, 1.0), (0.0, 1.0)]


simple uv unwrapped cube

faces:
[[0, 2, 3, 1], [4, 6, 7, 5], [8, 10, 11, 9], [12, 14, 15, 13], [22, 23, 20, 21], [16, 19, 18, 17]]
coords:
[(0.75, 0.5), (1.0, 0.5), (0.75, 0.2499999850988388), (1.0, 0.2499999850988388), (0.75, 0.2499999850988388), (0.75, -1.3212737570711397e-09), (0.4999999701976776, 0.2499999850988388), (0.4999999701976776, -1.3212737570711397e-09), (0.4999999701976776, 0.2499999850988388), (0.2499999701976776, 0.2499999850988388), (0.4999999701976776, 0.5), (0.2499999701976776, 0.5), (0.4999999701976776, 0.5), (0.4999999701976776, 0.75), ...]
```